experiments/chat/chat/views.py

```diff
@@ -25,12 +30,26 @@ class MessageCreate(CreateView):
     fields = ["text"]
 
     def get_success_url(self):
-        return reverse("detail", kwargs={"pk": self.kwargs["pk"]})
+        # Redirect to the empty form
+        return reverse("send", kwargs={"pk": self.kwargs["pk"]})
 
     def form_valid(self, form):
         room = get_object_or_404(Room, pk=self.kwargs["pk"])
         form.instance.room = room
-        super().form_valid(form)
-        return render(self.request, 'chat/message_update.html', {"message": form.instance}, content_type='text/html; turbo-stream;')
+        return super().form_valid(form)
+
+
+def wiretap(request):
```

Comment:
If we want to keep the "hotwire works without JS" thing (i personally do), the view here should display every message (or the N last ones). What do you think ?

Reply:
Agree although thats just a demo to show the alternative broadcast.